encoding/graph/type_/vertex.rs

```diff
@@ -69,8 +75,9 @@ impl Prefixed<BUFFER_KEY_INLINE> for TypeVertex {}
 
 impl Typed<BUFFER_KEY_INLINE> for TypeVertex {
     fn type_id_(&self) -> TypeID {
-        // TODO: is this the best way to get the TypeID out?
-        TypeID::decode(self.clone().to_bytes()[Self::RANGE_TYPE_ID].try_into().unwrap())
+        debug_assert_eq!(Self::LENGTH, 3);
+        debug_assert_eq!(Self::RANGE_TYPE_ID, 1..3);
+        TypeID { value: (self.value & 0xFFFF) as u16 }
```

Comment:
No.

Reply:
Technically `& 0xFFFF` is unnecessary, `as` is defined to truncate, but I decided to be explicit.